modules/reindex/src/internalClusterTest/java/org/opensearch/index/codec/MultiCodecReindexIT.java

```diff
@@ -61,72 +57,94 @@ public void testReindexingMultipleCodecs() throws InterruptedException, Executio
             "lz4",
             "BEST_SPEED"
         );
-
-        for (Map.Entry<String, String> codec : codecMap.entrySet()) {
-            assertReindexingWithMultipleCodecs(codec.getKey(), codec.getValue(), codecMap);
```

Comment:
Please reinstate the creation of index with each codec, it helps us verify that there is no break in the create index flow for each codec.

Reply:
Cool. I'll modify it a lil bit to create index with a random codec.